### ISSUE TITLE
Nulecule deployment is not complete

### ISSUE THREAD

**Issue by TomasTomecek** [projectatomic/atomic-site#274]
[Guide](https://github.com/projectatomic/atomic-site/blob/ff8bf80d51807a70b25e686e3c71d953a3771915/BUILD.md) mentions that I should

> configure answers.conf by copying Nulecule/answers.conf.sample to answers.conf

but there is no `answers.conf.sample`:

```
$ ll Nulecule/answers.conf.sample
ls: cannot access 'Nulecule/answers.conf.sample': No such file or directory
$ pwd
/home/tt/g/atomic-site
```

**Comment by jberkus** [comment]
oh, thanks!  Need to update the docs.

**Comment by jberkus** [comment]
Please check #282, see if it solves the issues.

**Comment by jberkus** [comment]
closed with merge of #282
